overrides-c9s.yaml

```diff
@@ -3,8 +3,13 @@
 # we need in the `packages` section. When not needed. Empty or comment out this
 # file (except this comment).
 
-#repos:
-#  - c9s-baseos-mirror
-#  - c9s-appstream-mirror
+repos:
+# - c9s-baseos-mirror
+  - c9s-appstream-mirror
 
-#packages:
+packages:
```

Comment:
Can you add a comment referencing https://issues.redhat.com/browse/RHEL-61612 for the pinned packages?

Reply:
done.